app/helpers/host_helper/textual_summary.rb

```diff
@@ -198,6 +198,28 @@ def textual_network
     h
   end
 
+  def textual_cloud_networks
+    num   = @record.number_of(:cloud_networks)
+    h     = {:label => _('Cloud Networks'), :icon => "ff ff-cloud-network", :value => num}
+    if num.positive? && role_allows?(:feature => "cloud_network_show_list")
+      h[:title] = _("Show all Cloud Networks")
+      h[:explorer] = true
+      h[:link]  = url_for_only_path(:action => 'cloud_networks', :id => @record, :display => "cloud_networks")
+    end
+    h
+  end
+
+  def textual_cloud_subnets
+    num   = @record.number_of(:cloud_subnets)
```

Comment:
Does `Host` really have `cloud_subnets` ?

Reply:
Yes, it has.